Week2/homework/part2/queries.js

```diff
@@ -0,0 +1,60 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser2',
+  password: 'hyfpassword',
+  database: 'new_world',
+});
+
+const help = () => {
+  console.log(
+    '\nTo add trigger for Vietnam type <node . trigger> \nTo insert new language for Vietnam type <node . insert>\n'
+  );
+  connection.end();
+};
+
+const addTrigger = () => {
+  connection.connect();
+  connection.query(
+    `
+  DROP TRIGGER IF EXISTS language_check;
+  
+  DELIMITER //
```

Comment:
I am getting a syntax error if I run the command or I try to run the command via mysql